Tests/XCUITests/UrlBarTests.swift

```diff
@@ -0,0 +1,27 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+
+class UrlBarTests: BaseTestCase {
+    // https://testrail.stage.mozaws.net/index.php?/cases/view/2189026
+    func testNewTabUrlBar() {
+        // Visit any website and select the URL bar
+        navigator.openURL("https://www.Mozilla.org")
```

Comment:
Could we open a website from `localhost` instead? I will soon clean up the tests that calls external websites. 🧹

Reply:
Updated